codesign/codesign.go

```diff
@@ -223,7 +223,7 @@ func (m *Manager) selectCodeSigningStrategy(credentials appleauth.Credentials) (
 	}
 
 	if isManaged {
-		return codeSigningXcode, "Using Xcode-managed code signing, as Automatically managed signing is enabled in Xcode for the project", nil
+		return codeSigningXcode, "Automatically managed signing is enabled in Xcode for the project", nil
```

Comment:
Usage of "Using..." prefix is not consistent, here was removed, other places not.